core/input/input_event.cpp

```diff
@@ -603,6 +623,10 @@ void InputEventKey::_bind_methods() {
 	ADD_PROPERTY(PropertyInfo(Variant::BOOL, "echo"), "set_echo", "is_echo");
 }
 
+InputEventKey::InputEventKey() {
+	event_state = InputEventState::RELEASED;
```

Comment:
Why not an initializer in the class declaration?

Reply:
`event_state` is a protected field from the base `InputEvent` class so we can't initialize it in `InputEventKey` class declaration.